src/libsync/propagateremotemkdir.h

```diff
@@ -59,6 +59,9 @@ private slots:
     void success();
 
 private:
-    bool _isEncryptedRootFolder = false;
+    void finalizeMkColJob(QNetworkReply::NetworkError err, const QString &jobHttpReasonPhraseString, const QString &jobPath);
+
+private:
+    QPair<SyncFileItem::Status, QString> _status = { SyncFileItem::NoStatus, QString() };
```

Comment:
This looks unused to me.

Reply:
@er-vin Not sure I follow. Could you elaborate on what does "unused" mean in the current context?

Reply:
@er-vin ahh nevermind, I got it now